src/HttpTracer/HttpTracer.csproj

```diff
@@ -33,4 +33,13 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
You don't need SDK extras for netstandard targets - you only want it when you go outside of that box.

Reply:
I thought that too. I added it back because I couldn't build it in Visual Studio for Mac 2017 or Rider without it.

Reply:
And I still refuse to install Visual Studio for Mac 2019, you know, due to the fact that it borders on being a dumpster fire 🔥.